src/browser/deck/actions.js

```diff
@@ -196,6 +196,20 @@ define((require, exports, module) => {
 
   const prepend = (items, item) => insert(items, item, 0);
 
+  const maybeSelectIndex = (items, index) => {
```

Comment:
Since you do activate a tab, please do not call this, `*select*`, but `*activate*` instead.